src/ImageSharp/Processing/Processors/Effects/OilPaintingProcessor{TPixel}.cs

```diff
@@ -97,19 +98,19 @@ public void Invoke(in RowInterval rows)
              * are cleared for each loop iteration, to avoid the repeated allocation for each processed pixel. */
             using IMemoryOwner<Vector4> sourceRowBuffer = this.configuration.MemoryAllocator.Allocate<Vector4>(this.source.Width);
             using IMemoryOwner<Vector4> targetRowBuffer = this.configuration.MemoryAllocator.Allocate<Vector4>(this.source.Width);
-            using IMemoryOwner<float> bins = this.configuration.MemoryAllocator.Allocate<float>(this.levels * 4);
+            using IMemoryOwner<float> bins = this.configuration.MemoryAllocator.Allocate<float>(this.levels * 256 * 4);
```

Comment:
Shouldn't we be clamping this in the extension method? I'm trying to wrap my head around how this actually works again.